hoot-core/src/main/cpp/hoot/core/io/OsmApiChangeset.h

```diff
@@ -360,7 +386,7 @@ class ChangesetInfo
   typedef typename container::iterator iterator;
   typedef typename container::const_iterator const_iterator;
   /** Constructor */
-  ChangesetInfo() { }
+  ChangesetInfo() : _changesetIssuesResolved(false) { }
```

Comment:
Don't know how long this has been a problem but this was never defaulted to `false` so the resolving code wasn't being called.